sei-tendermint/types/block.go

```diff
@@ -845,21 +845,34 @@ func (commit *Commit) ValidateBasic() error {
 	return nil
 }
 
-// Hash returns the hash of the commit
+// Hash returns the hash of the commit.
+// It computes a Merkle tree from all commit fields: Height, Round, BlockID, and Signatures.
```

Comment:
nit: we're not computing a merkle tree here? just concating the fields to the hash